backend/app/domains/users/crud.py

```
@@ -0,0 +1,87 @@
+    def create_student_profile(self, user: User, student_data: StudentCreate) -> Optional[Student]:
+        try:
+            student = Student(
+                user_id=user.id,
+                name=student_data.name,
+                email=user.email
+            )
+            self.session.add(student)
+            self.session.commit()
+            self.session.refresh(student)
+            return student
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)